process-automation/latest/model/connectors/outlook.md

```diff
@@ -16,32 +16,28 @@ The actions that can be executed using the Outlook connector are:
 
 ## Configuration
 
-For the Outlook connector to function specifc properties must be defined in the `application.properties` file of the Spring Boot application. By default, these properties are set to environment variables.
-
-The following is a set of standard properties that define the use of the connector:
-
-```bash
-teams.connector.client.id=${TEAMS_CLIENT_ID}
-teams.connector.client.secret=${TEAMS_CLIENT_SECRET}
-teams.connector.scope=${TEAMS_SCOPE}
-teams.connector.username=${TEAMS_USERNAME}
-teams.connector.password=${TEAMS_PASSWORD}
-teams.connector.tenant=${TEAMS_TENANT}
-```
-
-The connector uses a stream mechanism to send and receive information between Process Services and the connector. For this to occur you need to configure the following properties so the connector can be identified:
-
-```bash
-spring.cloud.stream.bindings.createTeamsEventCalendar.destination=${teams.connector.name}${spring.application.name}.CREATE_CALENDAR_EVENT
-spring.cloud.stream.bindings.createTeamsEventCalendar.contentType=application/json
-spring.cloud.stream.bindings.createTeamsEventCalendar.group=${CONSUMER_GROUP:${spring.application.name}}
-spring.cloud.stream.bindings.getScheduleAvailability.destination=${teams.connector.name}${spring.application.name}.GET_SCHEDULE_AVAILABILITY
-spring.cloud.stream.bindings.getScheduleAvailability.contentType=application/json
-spring.cloud.stream.bindings.getScheduleAvailability.group=${CONSUMER_GROUP:${spring.application.name}}
-spring.cloud.stream.bindings.updateTeamsEventCalendar.destination=${teams.connector.name}${spring.application.name}.UPDATE_CALENDAR_EVENT
-spring.cloud.stream.bindings.updateTeamsEventCalendar.contentType=application/json
-spring.cloud.stream.bindings.updateTeamsEventCalendar.group=${CONSUMER_GROUP:${spring.application.name}}
-```
+### Outlook connector configuration parameters
+
+The configuration parameters for the Outlook connector are:
+
+| Parameter | Description |
+|-----------|-------------|
+| TEAMS_CLIENT_ID | *Required.* The client identifier to be used for authentication. |
+| TEAMS_CLIENT_SECRET | *Required.* The client secret to be used for authentication. |
```

Comment:
Same here. Either use the article or don't use it consistently in this table.

Reply:
fixed.